libs/barista-components/tree-table/src/tree-table-toggle-cell.html

```diff
@@ -13,7 +13,5 @@
       [name]="_isExpanded ? 'dropdownopen' : 'dropdownclosed'"
     ></dt-icon>
   </button>
-  <div class="dt-tree-table-content">
-    <ng-content></ng-content>
-  </div>
+  <ng-content></ng-content>
```

Comment:
this was not a change from: https://github.com/dynatrace-oss/barista/pull/1564/files is this intended?